src/main/java/org/snomed/snowstorm/ecl/ECLQueryService.java

```diff
@@ -39,8 +43,15 @@
 	}
 
 	public Page<Long> selectConceptIds(String ecl, BranchCriteria branchCriteria, String path, boolean stated, Collection<Long> conceptIdFilter, PageRequest pageRequest) throws ECLException {
+		return selectRelevantConceptIds(ecl, branchCriteria, path, stated, conceptIdFilter, pageRequest, eclEdgeCaseHandlerService.replaceIncorrectConcreteAttributeValue(ecl, path, pageRequest));
```

Comment:
Consider putting the `replaceIncorrectConcreteAttributeValue` call on a separate line for readability. This is a significant step in processing. This style would be fine for some sort of utility method call.